### PR TITLE
lightning: check-only-doc

### PR DESCRIPTION
**Pull request by D3Hunter**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add `--check-only` flag to tidb-lighning

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): [code pr](https://github.com/pingcap/tidb/pull/32275)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@D3Hunter Is `check-only` related to the v6.0 release? Or to a future release?

**Review by sunzhaoyang — APPROVED** [review]
LGTM

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -424,6 +424,7 @@ min-available-ratio = 0.05
 | --cert *file* | TLS 连接的证书路径 | `security.cert-path` |
 | --key *file* | TLS 连接的私钥路径 | `security.key-path` |
 | --server-mode | 在服务器模式下启动 TiDB Lightning | `lightning.server-mode` |
+| --check-only | 仅进行前置检查和数据采样检查，数据采样检查会对一定比例的数据文件的前 N 行进行数据检查，主要的检查项为：数据行数是否与列数匹配，字符类型列是否存在非法字符。<br/>可选择值为 `--check-only=default`，此时数据采样检查文件比例为`1%`，每个文件检查前 `1000` 行；或 `--check-only=rate,rows`，其中 `rate` 是`(0, 1]`范围内的浮点数，代表文件采样的比例，`rows` 为正整数或`-1`，代表每个采样的文件需检查的行数，`-1` 代表对文件的所有行进行检查。<br/>v6.0 版本新增。 | 仅在命令行生效，无对应配置项 |
```
Comment (en-jin19) [inline tidb-lightning/tidb-lightning-configuration.md:427]:
```suggestion
| --check-only | 仅进行前置检查和数据采样检查（从 v6.0 版本开始引入）。数据采样检查是【XXX什么】 对 【XXX在哪里】中一定比例的数据文件的前 N 行进行的数据检查，主要用于检查数据行数是否与列数匹配、字符类型列是否存在非法字符等等。<br/>该参数的可选值为 `default` 和 `rate,rows`。当设置为 `--check-only=default` 时，待进行数据采样检查的文件比例为 `1%`，待检查行数为 `1000` 行（即【XXX什么】 会对【XXX哪里】中 1% 的文件进行数据采样检查，并检查每个文件的前 1000 行）。在可选值 `rate,rows` 中，`rate` 为 (0, 1] 范围内的浮点数，代表文件采样的比例；`rows` 为正整数或 `-1`，代表每个采样的文件需检查的行数。当 `rows` 为 `-1` 时，表示对文件的所有行进行检查。比如，当 【`--check-only=XXX,-1`】时，【XXX什么】 会对 【XXX哪里】中 【XXX%】 的文件进行数据采样检查，并检查每个文件的所有行。 | 无对应配置项（仅在命令行中生效） |
```

Comment (en-jin19) [reply tidb-lightning/tidb-lightning-configuration.md:427]: @D3Hunter 有一些不确定的内容使用 【XXX】 的方式标记了，可以请帮忙看下需要如何填写这些部分吗？

另外，文中没有介绍如何通过该参数进行前置检查，可以请补充相关内容吗？谢谢！

Comment (D3Hunter) [reply tidb-lightning/tidb-lightning-configuration.md:427]: 开了这个参数，就会进行前置检查和抽样检查，这俩一块做

**Comment by D3Hunter** [comment]
@en-jin19 ptal

**Pull request closed without merging**